ToolBox/Program.cs

```diff
@@ -55,12 +57,15 @@ public class Program
         public static void Main(string[] args)
         {
             var optionsObject = ToolboxArgumentParser.ParseArguments(args);
+
+            // if no console args supplied check if config.json has working info
             if (optionsObject.Count == 0)
             {
-                PrintMessageAndExit();
+                optionsObject = LoadOptionsObjectFromConfigurationFile("config.json");
```

Comment:
We'll want to make sure we update the documentation to be explicit about this behavior. Upon first reading of the PR I was thinking that `config.json` would be used to resolve any values not specified, which would permit a mixed-mode, some via cmd line and some via config. It looks like the way it's set up is all or nothing.

Reply:
That's correct, for those who prefer to do all the settings through config file - they can use only config. and launch from visual studio . Passing no console argument . For those who choose to use a console things will work as before.